src/main.cpp

```diff
@@ -27,10 +27,11 @@ int main() {
     sp->load_tools();
     delete sp;
 
+    // TOADDBACK 
     // Create all TemperatureControl modules. Note order is important here must be after extruder so Tn as a parameter will get executed first
-    TemperatureControlPool *tp= new TemperatureControlPool();
-    tp->load_tools();
-    delete tp;
+    // TemperatureControlPool *tp= new TemperatureControlPool();

```

Comment:
I saw a crash when the SlowTicker would attempt to read an uninitialized Adc object to get temperature. Is this fix you made here related to the same crash?